# Copyright (C) 2013      Nick Stenning <nick@whiteink.com>
# Copyright (C) 2014-2017 Maciej Delmanowski <drybjed@gmail.com>
# Copyright (C) 2014-2017 DebOps <https://debops.org/>
# SPDX-License-Identifier: GPL-2.0-only
# The original repository does not have a license. The license was chosen based
# on the license of the Mailman 2 codebase.

# Add direct link to moderation page
# https://github.com/okfn/infra/commit/06b83759238e38d1b239ee1e04d75ae3e46365ae

--- Mailman/HTMLFormatter.py
+++ Mailman/HTMLFormatter.py
@@ -56,6 +56,10 @@
                          _('%(realname)s administrative interface')),
                     _(' (requires authorization)'),
                     '<br>',
+                    Link(self.GetScriptURL('admindb'),
+                         _('%(realname)s moderation interface')),
+                    _(' (requires authorization)'),
+                    '<br>',
                     Link(Utils.ScriptURL('listinfo'),
                          _('Overview of all %(hostname)s mailing lists')),
                     '<p>', MailmanLogo()))).Format()
